src/main/java/org/sonatype/repository/helm/internal/util/HelmAttributeParser.java

```diff
@@ -57,11 +57,18 @@ public HelmAttributes getAttributesFromInputStream(final InputStream inputStream
       HelmAttributes helmAttributes = new HelmAttributes();
       helmAttributes.setName(attributes.get(NAME).toString());
       helmAttributes.setVersion(attributes.get(VERSION).toString());
-      helmAttributes.setDescription(attributes.get(DESCRIPTION).toString());
-      helmAttributes.setIcon(attributes.get(ICON).toString());
-      helmAttributes.setMaintainers((List<Map<String, String>>)attributes.get(MAINTAINERS));
-      helmAttributes.setSources((List<String>)attributes.get(SOURCES));
-
+      if (attributes.get(DESCRIPTION) != null) {
```

Comment:
This is great, but you could maybe do `Objects.nonNull(attributes.get(DESCRIPTION))` or `null != attributes.get(DESCRIPTION)` (both avoid assign by typo errors). Up to you however!

Reply:
Sounds good. Sorry, I’m a C, Ruby, python developer.  Haven’t really done much java in years.  😂

Reply:
What you did is fine, those two methods I showed just make it harder to
make a typo error over time, the jury is still out on if they are more
readable though :)

Also kinda moot point given you can’t make an assignment on the check you
are doing, just good practice overall

Reply:
@dtshepherd could you please also add support for appVersion field.
      if (attributes.get(APP_VERSION) != null) {
        helmAttributes.setAppVersion(attributes.get(APP_VERSION).toString());
      }
Thank you.